console/src/ontology/Tree.tsx

```diff
@@ -183,6 +183,12 @@ const handleRelationshipsChange = async (
     setNodes([...nextTree]);
   });
 
+const sortFunc = (a: Core.Node, b: Core.Node) => {
+  if (a.key.startsWith("group")) return -1;
```

Comment:
If `a` and `b` are both groups we should use default sort. Also, pull in the group ontology type from the client